Dockerfile

```diff
@@ -27,20 +27,19 @@ RUN apt-get install -y mafft && \
     mafft --version
     
 RUN apt-get install -y fasttree && \
-    ln -s /usr/bin/fasttree /usr/bin/FastTree && \
     FastTree -expert
 
 RUN mkdir -p /kb/deps
 WORKDIR /kb/deps
 
 RUN wget --quiet https://github.com/davidemms/OrthoFinder/releases/download/2.5.2/OrthoFinder_source.tar.gz && \
```

Comment:
RUN \
    # get the debian deps
    apt-get update && apt-get upgrade -y && \
    DEBIAN_FRONTEND=noninteractive apt-get install -y wget grace mcl mafft fasttree && \
    mafft --version && \
    FastTree -expert && \
    # retrieve phantomjs (does one of the dependencies use it? it's usually just used for testing javascript)
    conda install -c conda-forge phantomjs && \
    # install python deps
    pip install --upgrade pip && pip install scipy bokeh selenium && \
    # create a dependencies dir and download the various other bits and pieces into it
    mkdir -p /kb/deps && cd /kb/deps && \
    wget --quiet https://github.com/bbuchfink/diamond/releases/download/v2.0.9/diamond-linux64.tar.gz && \
    tar -zxf diamond-linux64.tar.gz diamond && \
    mv diamond /usr/bin/diamond && \
    diamond version && \
    wget --quiet https://github.com/davidemms/OrthoFinder/releases/download/2.5.2/OrthoFinder_source.tar.gz && \
    tar -zxf OrthoFinder_source.tar.gz && \
    mv OrthoFinder_source /usr/bin/orthofinder && \
    # Using a fork with fixes for Python 3
    git clone https://github.com/samseaver/pygrace.git && \
    cd pygrace && \
    mv PyGrace /opt/conda3/lib/python3.8/site-packages/PyGrace && \
    cd .. && \
    # Loading PlantSEED data
    git clone -b kbase_release https://github.com/ModelSEED/PlantSEED /kb/module/PlantSEED
# -----------------------------------------
COPY ./ /kb/module
WORKDIR /kb/module
RUN mkdir -p /kb/module/work && \
    chmod -R a+rw /kb/module && \
    make all && \
    # remove the detritus from /kb/deps
    rm -r /kb/deps

ENTRYPOINT [ "./scripts/entrypoint.sh" ]